src/scripts/create_ap

```diff
@@ -62,6 +62,7 @@ usage() {
     echo "                          back to managed"
     echo "  --mac <MAC>             Set MAC address"
     echo "  --dhcp-dns <IP1[,IP2]>  Set DNS returned by DHCP"
+    echo "  --dhcp-range <range>    Set the DHCP range. Valid values are between 1 and 254. (example: 150,160)"
```

Comment:
Better replace `<range>` with `<start,end>` to make it clear it needs 2 numbers and their purpose...
It might even be better to split this option into two options for more flexibility and purpose:
1. `--dhcp-start` and
2. `--dhcp-end`